src/style/content/conversation/message-list.less

```diff
@@ -383,7 +383,6 @@
   position: relative;
   display: flex;
   max-width: var(--conversation-message-asset-width);
-  margin: 0 0 24px;
```

Comment:
the bottom margin of 24px for image is a left over from the original (8 years old) commit.
All assets now share the same padding/margin